data_collection/gazette/spiders/sp/sp_cacapava.py

```diff
@@ -31,12 +31,13 @@ def parse(self, response):
             edition_number = gazette.css("h3::text").re_first(r"Edição nº (\d+)")
             gazette_raw_date = gazette.css("p::text").re_first(r"\d{2}/\d{2}/\d{4}")
             gazette_date = datetime.strptime(gazette_raw_date, "%d/%m/%Y").date()
-            gazette_url = gazette.css("a::attr(href)")
+            gazette_url = gazette.css("a::attr(href)").extract()
 
             yield Gazette(
                 date=gazette_date,
                 edition_number=edition_number,
                 is_extra_edition=False,
                 power="executive_legislative",
-                file_urls=[gazette_url],
+                territory_id=self.TERRITORY_ID,
+                file_urls=gazette_url,
```

Comment:
@jjpaulo2 uam dúvida , essa url exportada é uma lista? pois pelo que eu vi o pessoal utiliza geralmente uma lista nessa aprte, acredito que se tiver uma o adeqaudo seria fazer a mudança abaixo
```suggestion
                file_urls=[gazette_url],
```

Reply:
Nesse caso, o objeto já é uma lista. Fiz o evalutate com `.extract()`. 

Vi que a maioria dos spiders usa sempre `.extract_first()` (esse só retorna um registro). Caso seja o padrão, ou o recomendável no projeto, posso alterar sem problemas.

Reply:
Show de bola, vou marcar como resolvido então já que se trata de uma lista